AdyenDropIn/Models/DropInConfiguration.swift

```diff
@@ -253,5 +253,19 @@ public extension DropInComponent {
             self.storePaymentMethod = storePaymentMethod
         }
     }
-    
+
+    /// Twint component configuration.
+    struct Twint: AnyTwintComponentConfiguration {
```

Comment:
I don't see this is added to DropIn's configuration as a property yet right? Needs to be added there and then passed on to the TwintComponentConfiguration so this could be set via dropin

Reply:
Added!